packages/caliper-fabric/test/sample-configs/Org2StaticConnProfile.yaml

```diff
@@ -0,0 +1,115 @@
+---
```

Comment:
Should be called Org2StaticConnectionProfile

We try to avoid using abbreviations in code and filenames